clients/ui/frontend/src/__tests__/cypress/cypress/tests/mocked/modelRegistry/modelVersions.cy.ts

```diff
@@ -251,7 +251,7 @@ describe('Model Versions', () => {
     const modelVersionRow = modelRegistry.getModelVersionRow('model version');
     modelVersionRow.findModelVersionName().contains('model version').click();
     verifyRelativeURL('/model-registry/modelregistry-sample/registeredModels/1/versions/1/details');
-    cy.findByTestId('app-page-title').should('have.text', 'model version');
+    cy.findByTestId('app-page-title').should('contain.text', 'test');
```

Comment:
add another line for checking for the model name along with the model version dropdown